bd_spot_wrapper/spot_wrapper/april_tag_pose_estimator.py

```diff
@@ -0,0 +1,101 @@
+from typing import List, Tuple
+
+import cv2.aruco as aruco
+import fairotag as frt
```

Comment:
Do we need to update readme for this? And we do `git submodule` for `fairotag`?

Reply:
I am not sure if it should be a sub-module or straight-away `pip install -e .` 
I have update the setup instructions for this, can you please take a look at it when you have a chance?